packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7917,6 +7926,33 @@ export default class Meeting extends StatelessWebexPlugin {
       });
   }
 
+  /**
+   * set sending named media group which the audio should send to
+   * @param {MediaType} mediaType of the stream
+   * @param {number} languageCode of the stream
+   * @returns {void}
+   */
+  public setSendNamedMediaGroup(mediaType: MediaType, languageCode = 0): void {
+    if (mediaType !== MediaType.AudioMain) {
+      throw new Error(`cannot set send named media group which media type is ${mediaType}`);
+    }
+
+    const value = languageCode || this.simultaneousInterpretation.getTargetLanguageCode();
+    let groups = [];
+
+    if (value) {
+      groups = [
+        {
+          type: 1,
```

Comment:
is there some enum or const we can use here instead of "1"?

Reply:
Done